--- build/android-x86_64/Makefile	2019-06-07 11:54:43.000000000 +0200
+++ build/android-x86_64/Makefile_new	2019-06-07 11:55:08.000000000 +0200
@@ -63,1 +63,1 @@
-CFLAG= -DOPENSSL_THREADS -D_REENTRANT -DDSO_DLFCN -DHAVE_DLFCN_H -DOPENSSL_PIC -fPIC -mandroid -I$(ANDROID_DEV)/include -B$(ANDROID_DEV)/lib -O3 -fomit-frame-pointer -Wall
+CFLAG= -DOPENSSL_THREADS -D_REENTRANT -DDSO_DLFCN -DHAVE_DLFCN_H -DOPENSSL_PIC -fPIC -I$(ANDROID_DEV)/include -B$(ANDROID_DEV)/lib -O3 -fomit-frame-pointer -Wall
